### ISSUE TITLE
[1.18.1] Suggestion: Zombie Arms/Shiny Zombie Arm should be a chance to drop with Looting, not guaranteed

### ISSUE THREAD

**Issue by Not-February** [stal111/Forbidden-Arcanus#238]
I get so many zombie arms just from killing things regularly that I now go out of my way to *avoid* killing zombies and drowned whenever possible, just because them dropping an arm every time I kill them is way too much inventory clutter - especially considering they don't seem to take damage when used as a weapon.

**Comment by stal111** [comment]
A Zombie has a 1/500 chance to drop a Zombie Arm, not on every kill. Have you modified the loot table somehow? I just tested with only f&a and it's dropping really rarely as it should so it isn't caused by f&a itself. It might also be some mod conflict in case you are playing a modpack.

**Comment by Not-February** [comment]
I tested on a hunch and I've discovered that while my issue was partially due to the involvement of another mod, it's actually something related to looting of any level.

In my modpack I was wearing Enigmatic Legacy's Ring of Seven Curses, which among other things gives you an additional level of looting and fortune and cannot be taken off once put on. Wearing the ring and killing a zombie drops an arm every single time.

But then I tested in creative *without* the ring, and if there is *any* level of looting on a weapon you use to kill a zombie, the zombie will always drop an arm, even if it's only Looting I.

Killing a zombie with no looting of any kind doesn't drop an arm every time.

Looking at the loot tables for F&A on Github I'm wondering if it's because there's no looting modifier in the loot pool additions, so anytime looting is applied it just drops it every time instead of adding a multiplier like other rare loot in Minecraft does afaik. It was looking at these tables and seeing no modifier for drop rates at all which led me to initially assume that it was just always dropping, which was an error on my part.
